docker-compose.yml

```
@@ -0,0 +1,24 @@
+services:
+    dev:
+        build:
+            context: .
+            target: dev
+        image: todo-app:dev
+        container_name: todo-app.dev
+        ports: 
+            - "5000:5000"
+        env_file: .env
+        volumes:
+            - type: bind
+              source: todo_app
+              target: /opt/todoapp/todo_app
```

Many people prefer this clearer syntax, but in case it's of interest this is equivalent to simply:
```suggestion
            - ./todo_app:/opt/todoapp/todo_app
```
The relative path is enough to be clear it's a bind mount, and then we use the `:` to separate source and target